docs/source/guide/general/client.rst

```diff
@@ -0,0 +1,84 @@
+##########
+TUF Client
+##########
+
+RSTUF makes it easy to host and manage TUF metadata for your content
+repository.
+
+In order to effectively protect your clients, they must download or update
+artifacts from your content repository along with metadata served by RSTUF,
+and interpret them in a TUF-compliant way.
+
+.. uml::
+
+    @startuml
+        actor User as User
+        User -> Client: Download artifact vX.Y.Z
+        group TUF Client
+            collections TUF as "TUF Metadata"
+            Client -> TUF: Request metadata
+
+            Artifact --> Client: Fetch artifact vX.Y.Z
+        end
+        Client -> User: Artifact vX.Y.Z
+    @enduml
+
+
+RSTUF itself does not provide client-side tools, but there are several TUF
+libraries that you can use.
+
+TUF client implementation
+=========================
+
+Depending on whether you already have an existing client application the
+following options exist.
+
+New client
+----------
+
+If you don't have a client implementation, you should consider using
+python-tuf's `ngclient <https://theupdateframework.readthedocs.io/en/latest/api/tuf.ngclient.html>`_.
+It can download and validate metadata and artifacts in a fully specification-compliant way.
+Moreover, RSTUF is built with python-tuf and guaranteed to be compatible with *ngclient*. See
+`client example application <https://github.com/theupdateframework/python-tuf/tree/develop/examples/client>`_
+to get started.
```

Comment:
```suggestion
To establish a new client implementation, consider using
python-tuf's `ngclient <https://theupdateframework.readthedocs.io/en/latest/api/tuf.ngclient.html>`_.
It can download and validate metadata and artifacts in a fully specification-compliant way.
Moreover, RSTUF is built with python-tuf and guaranteed to be compatible with *ngclient*. See
`client example application <https://github.com/theupdateframework/python-tuf/tree/develop/examples/client>`_
to get started.
```